sdks/python/apache_beam/runners/common.py

```diff
@@ -59,12 +59,16 @@ def __init__(self,
                kwargs,
                side_inputs,
                windowing,
-               context,
-               tagged_receivers,
+               context=None,
+               tagged_receivers=None,
                logger=None,
                step_name=None,
                # Preferred alternative to logger
-               logging_context=None):
+               # TODO(robertwb): Remove once all runners are updated.
```

Comment:
What will be removed when all runners are updated? logging_context and state? Or context parameter? Or the None defaults?